### ISSUE TITLE
[Bug] Kopernicus interop (v0.04 dev)

### ISSUE THREAD

**Issue by PhineasFreak** [LGhassen/Scatterer#49]
**Description:**

The Scatterer effects do not appear in any of the KSP scenes (Space Center, Tracking Station, Map & Flight View) when Kopernicus is installed. Upon Kopernicus unistallation everything works as expected. Happens in all renderer options (DX9/DX11/OpenGL) with just Kopernicus (along with it's MM and MFI dependencies) installed.

**Notes:**

An excerpt from the KSP log file reveals the following NullReference when Scatterer Core is initialized:

```
[EXC 16:02:47.437] NullReferenceException: Object reference not set to an instance of an object
	scatterer.Core.Init ()
	scatterer.Core+<DelayedInit>d__3.MoveNext ()
	UnityEngine.SetupCoroutine.InvokeMoveNext (IEnumerator enumerator, IntPtr returnValueAddress)
```

After i inserted some debug log lines in the **Init()** routine (**Core.cs**) i pinpointed the error around the Sun corona RenderQueue patcher (lines **[432-438](https://github.com/LGhassen/Scatterer/blob/d7177645fbce54fb2a0896e94ea876e76ae95547/scatterer/Core.cs#L432-L438)**). Temporarily commenting out these lines and recompiling the assembly was enough to allow the Scatterer effects to load/appear (as expected, the Sun flares did not appear).

**[KSP log file link](https://raw.githubusercontent.com/PhineasFreak/DocBin/master/DocBin/log_files/scatterer_20181128_140138_log.txt)**

**Test Environment:**

Tested in a new **KSP 1.5.1.2335** installation with just the following mods installed:

* **Kopernicus 1.5.1-1**
* **Modular Flight Integrator 1.2.6**
* **Scatterer 0.04 Dev**

**Comment by LGhassen** [comment]
Thanks for the detailed report.

**Comment by LGhassen** [comment]
Should be fixed in commit 2686d9c @PhineasFreak  can you retest?

**Comment by PhineasFreak** [comment]
Tested the new assembly with the same mod list and everything operates as expected. Thank you for the quick action!
